reward/stakingManager.go

```diff
@@ -0,0 +1,101 @@
+package reward
+
+import (
+	"github.com/klaytn/klaytn/blockchain"
+	"github.com/klaytn/klaytn/event"
+	"github.com/klaytn/klaytn/params"
+)
+
+const (
+	chainHeadChanSize = 100
```

Comment:
Just a question:
Is `100` enough to handle the event for the heavy loads?

Reply:
yes it is. Because we use chainHead event, it will be okay even in the heavy loads